### ISSUE TITLE
Alternative text Provided for the "Chart" is not descriptive enough: A11y_.NET 9 template_.NET 9 template_Chart_Non text content.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26260]
### GitHub Tags
#A11yTCS; #A11ySev3; #Win32; #WCAG1.1.1; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #
NonTextContent; #A11yMAS;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Observe that Alternative text Provided for the "Chart" is not descriptive enough.

### Actual Result:
Alternative text Provided for the "Chart" is not descriptive enough.

### Expected Result:
Alternative text Provided for the "Chart" should be more descriptive so will understand it's purpose.
### 
### User Impact:
All users will get impacted if the Alternative text is not descriptive.

### Attachment:

https://github.com/user-attachments/assets/16ba7eaa-a22c-4131-a9df-312fe4684227

**Comment by sheiksyedm** [comment]
These changes are done in the below PR:
https://github.com/dotnet/maui-samples/pull/569

**Comment by AnKushSingh05** [comment]
Issue is fixed, Hence closing the bug

![Image](https://github.com/user-attachments/assets/65491665-84d5-402f-9a1a-3de37bc3f270)
